cmd/dosa/main.go

```diff
@@ -42,6 +43,8 @@ var (
 	version   = "0.0.0"
 	githash   = "master"
 	timestamp = "now"
+	javaclientVersion = "2.0.0-beta"
```

Comment:
How can the version because 2.0.0??? we haven't ever released anything yet.

Reply:
Changed to 1.1.0-beta